frontend/public/components/sidebars/resource-sidebar.jsx

```diff
@@ -60,44 +65,35 @@ class ResourceSidebarWrapper extends React.Component {
   }
 }
 
-export const SampleYaml = ({ sample, loadSampleYaml, downloadSampleYaml }) => {
-  const { highlightText, header, subheader, img, details, templateName, kind } = sample;
-  return (
-    <li className="co-resource-sidebar-item">
-      <h5 className="co-resource-sidebar-item__header">
-        <span className="text-uppercase">{highlightText}</span> {header}{' '}
-        <span className="co-role-sidebar-subheader">{subheader}</span>
-      </h5>
-      {img && <img src={img} className="co-resource-sidebar-item__img" />}
-      <p className="co-resource-sidebar-item__details">{details}</p>
-      <button className="btn btn-link" onClick={() => loadSampleYaml(templateName, kind)}>
```

Comment:
@sg00dwin: no need to migrate these buttons (note they've been moved to frontend/public/components/sidebars/resource-sidebar-samples.tsx)